src/visitors/hoist.js

```diff
@@ -330,7 +330,10 @@ module.exports = {
     } else if (declaration) {
       path.replaceWith(declaration);
 
-      let identifiers = t.getBindingIdentifiers(declaration);
+      let identifiers = t.isIdentifier(declaration.id)
+        ? [declaration.id]
+        : t.getBindingIdentifiers(declaration);
```

Comment:
Did this because `getBindingIdentifiers` also returns `params`, it caused arguments to be treated as exports